src/dndbeyond/content-scripts/character.js

```diff
@@ -865,7 +866,7 @@ async function rollItem(force_display = false, force_to_hit_only = false, force_
 
         to_hit = handleSpecialGeneralAttacks(damages, damage_types, properties, settings_to_change, {to_hit, item_name});
 
-        to_hit = handleSpecialWeaponAttacks(damages, damage_types, properties, settings_to_change, {item_customizations, item_type, to_hit});
+        to_hit = handleSpecialWeaponAttacks(damages, damage_types, properties, settings_to_change, {item_customizations, item_type, to_hit, is_infused});
```

Comment:
Humm... I'd do it a bit differently to keep the arguments to `handleSpecialWeaponAttacks` cleaner.
```js
if (is_infused) {
  properties["Infusion"] = "Yes"
}
```
Then check for the `properties["Infusion"] in the `handleSpecialWeaponAttacks`.
That'd be good for me.

Reply:
New code pushed, give it a look 😄